reg/drone/service/actuator/esc/Status.0.1.uavcan

```diff
@@ -0,0 +1,21 @@
+# Detailed ESC status published at a low rate.
+# The publication period should be consistent across the group.
+
+uint8 MAX_PUBLICATION_PERIOD = 1  # [second]
+
+uint32 error_count
+# Incremented once per occurrence. Reset to zero when started.
+# The exact definition of what constitutes an error is implementation-dependent.
+
+FaultFlags.0.1 fault_flags
+
+reg.drone.service.actuator.util.Feedback.0.1 feedback
+# The feedback is included in its entirety so that a subscriber interested in receiving full information
+# at a low rate (e.g., for monitoring) would not have to subscribe to both Feedback and Status separately.
+
+float16 motor_temperature   # [kelvin]
+float16 driver_temperature  # [kelvin]
+# Sampled temperatures. If multiple values are available, reduction is implementation-defined.
+# We use raw float16 here instead of the more type-safe SI types to reduce the footprint.
```

Comment:
We need to take into consideration the multi-frame parsing requirements introduced and the repercussions on legacy CAN.

Reply:
Do you have any specific action items for this?

Reply:
Can we expose that concern in `reg/drone/service/actuator/esc/_.0.1.uavcan` and mark it as something to validate?

Reply:
I wrote there that Classic CAN nodes should throttle the rate to 100 Hz.